src/sagemaker/model_card/model_card.py

```diff
@@ -863,6 +1395,7 @@ def __init__(
         training_details: Optional[TrainingDetails] = None,
         evaluation_details: Optional[List[EvaluationJob]] = None,
         additional_information: Optional[AdditionalInformation] = None,
+        model_package_details: Optional[ModelPackage] = None,
```

Comment:
if this is an existing feature, would prefer new param added to the end for backwards compatibility for users not using named arguments (but positional args) in their code.

Reply:
Yes, model card is an existing feature. Good catch. Thank you. I will move that to the end.